resources/public/api/conf/5.0/foreign_property_cumulative_summary_create_and_amend.yaml

```diff
@@ -19,7 +18,7 @@ description: |
   | 2025-26 onwards | OUTSIDE_AMENDMENT_WINDOW                    | Simulates the scenario where request cannot be completed as it is outside the amendment window. |
 
 tags:
-  - Foreign Property Cumulative Submission
+  - Foreign Property Cumulative Summary
```

Comment:
This and the retrieve one will also need updating once the tags are updated in the application.yaml